src/Hosting/Abstractions/src/WebHostDefaults.cs

```diff
@@ -21,5 +21,6 @@ public static class WebHostDefaults
         public static readonly string SuppressStatusMessagesKey = "suppressStatusMessages";
 
         public static readonly string ShutdownTimeoutKey = "shutdownTimeoutSeconds";
+        public static readonly string StaticWebAssetsKey = "staticwebassets";
```

Comment:
Tiny nit - others use camelcase:

```suggestion
        public static readonly string StaticWebAssetsKey = "staticWebAssets";
```